app/views/service_requests/add_service.js.coffee

```diff
@@ -22,7 +22,7 @@
 ConfirmSwal.fire(
   icon: 'question'
   title: I18n.t('proper.catalog.new_request.header')
-  text: I18n.t('proper.catalog.new_request.warning')
+  text: " ",
```

Comment:
Should this be left empty or just removed?

Reply:
It was left empty on purpose.. it's the only way I could figure how to match acceptance criteria. If the line is removed then SweetAlert auto-populates a default message at that sub-header node, but we don't want any text there.